### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dejurin**
Added instructions on how to deploy Cloudflare.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,3 +143,13 @@ For full PWA compatibility, you can put your [screenshots](https://developer.moz
 ```
 
 For beautiful screenshots, you can use [Progressier Screenshots Generator](https://progressier.com/pwa-screenshots-generator).
+
+### Cloudflare deployment
+
+**PNPM**
+1. Need set resolutions w/ `"sharp": "0.32.6"` on **package.json**
+2. It is necessary to create a file **npm-lock.yaml** by `pnpm install --no-frozen-lockfile`
```
Comment (Aslemammad) [inline README.md]: Let's say "re-run `pnpm install` to apply the changes" for point 2.

Would that work?

Comment (dejurin) [reply README.md]: I apologize, you are correct, only the `pnpm install` is sufficient here. The file **npm-lock.yaml** must exist.

Comment (Aslemammad) [reply README.md]: That'd be great, let me know. I'll merge this after you push the change.

Comment (dejurin) [reply README.md]: Im done.

**Comment by Aslemammad** [comment]
Thank you so much.

**Pull request merged**